modules/ROOT/pages/_partials/commons/common-logging.adoc

```diff
@@ -29,94 +29,178 @@ include::{root-partials}_block-abstract.adoc[]
 endif::[]
 // End -- Output an abstract unless this inclusion is being used as part of an encompassing page
 
+:logging-classes: {url-api-class-log}, {url-api-class-log-file} and {url-api-class-log-config}
+ifeval::["{param-platform}"=="{platform-net}"]
+:logging-classes: {url-api-class-logging}
+endif::[]
+
+ifdef::is-android[]
+.Constraints
+endif::[]
+
+[NOTE]
+--
+The retrieval of logs from the device is out of scope of this feature.
+
+ifdef::is-android[]
```

Comment:
FYI as of 3.0, it will disappear.  I believe it is already deprecated...

Reply:
Thanks